server/models/postgis/project.py

```diff
@@ -468,17 +468,10 @@ def get_project_stats(self) -> ProjectStatsDTO:
         """ Create Project Summary model for postgis project object"""
         project_stats = ProjectStatsDTO()
         project_stats.project_id = self.id
-        polygon = to_shape(self.geometry)
-        polygon_aea = transform(
-            partial(
-                pyproj.transform,
-                pyproj.Proj(init="EPSG:4326"),
-                pyproj.Proj(proj="aea", lat1=polygon.bounds[1], lat2=polygon.bounds[3]),
-            ),
-            polygon,
-        )
-        area = polygon_aea.area / 1000000
-        project_stats.area = area
+        project_area_sql = "select ST_Area(geometry, true)/1000000 as area from public.projects where id = :id"
```

Comment:
@ramyaragupathy I think we would need to cast the geom column as geography type in order to get the area in squared meters. Please check the third example from https://postgis.net/docs/ST_Area.html

Reply:
@JorgeMartinezG - I tried the examples with the power value mentioned in the documentation. The one in the PR now `ST_Area(geometry, true)/1000000` seems to be matching with the current turf value returned

Reply:
To me it seems accurate. The example @JorgeMartinezG was referring to seems to be in a different projection (EPSG:2249) in our case this query should return m2 or km2 after the division.